examples/all-clusters-app/cc13x4_26x4/main/include/AppConfig.h

```diff
@@ -1,6 +1,5 @@
 /*
  *    Copyright (c) 2020 Project CHIP Authors
- *    Copyright (c) 2019 Google LLC.
```

Comment:
This seems like an odd thing for someone who is not Google to remove.

Reply:
These copyright headers had been previously imported into the projects incorrectly, this PR aims to correct these earlier errors.